[2023-11-06T12:06:04.897+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-26T00:00:00+00:00 [queued]>
[2023-11-06T12:06:04.914+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-26T00:00:00+00:00 [queued]>
[2023-11-06T12:06:04.915+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:06:05.024+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-10-26 00:00:00+00:00
[2023-11-06T12:06:05.032+0000] {standard_task_runner.py:57} INFO - Started process 872 to run task
[2023-11-06T12:06:05.039+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-10-26T00:00:00+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmpvs20qa12']
[2023-11-06T12:06:05.046+0000] {standard_task_runner.py:85} INFO - Job 100: Subtask task_a
[2023-11-06T12:06:05.154+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-26T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:06:05.312+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-10-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-26T00:00:00+00:00'
[2023-11-06T12:06:05.314+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:06:05.316+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World :)']
[2023-11-06T12:06:05.331+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:06:05.338+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: syntax error near unexpected token `)'
[2023-11-06T12:06:05.339+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: `echo Hello World :)'
[2023-11-06T12:06:05.341+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-11-06T12:06:05.363+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-11-06T12:06:05.370+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231026T000000, start_date=20231106T120604, end_date=20231106T120605
[2023-11-06T12:06:05.412+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 100 for task task_a (Bash command failed. The command returned a non-zero exit code 2.; 872)
[2023-11-06T12:06:05.460+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-06T12:06:05.620+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-06T12:19:09.980+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-26T00:00:00+00:00 [queued]>
[2023-11-06T12:19:10.007+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-26T00:00:00+00:00 [queued]>
[2023-11-06T12:19:10.009+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:19:10.146+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-10-26 00:00:00+00:00
[2023-11-06T12:19:10.162+0000] {standard_task_runner.py:57} INFO - Started process 2083 to run task
[2023-11-06T12:19:10.170+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-10-26T00:00:00+00:00', '--job-id', '172', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmpyc5rcsdb']
[2023-11-06T12:19:10.179+0000] {standard_task_runner.py:85} INFO - Job 172: Subtask task_a
[2023-11-06T12:19:10.304+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-26T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:19:10.519+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-10-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-26T00:00:00+00:00'
[2023-11-06T12:19:10.529+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:19:10.532+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World.']
[2023-11-06T12:19:10.551+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:19:10.555+0000] {subprocess.py:93} INFO - Hello World.
[2023-11-06T12:19:10.557+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-11-06T12:19:10.607+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231026T000000, start_date=20231106T121909, end_date=20231106T121910
[2023-11-06T12:19:10.678+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-06T12:19:10.712+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
